src/shared/components/window/WindowStore.ts

```diff
@@ -0,0 +1,32 @@
+import {action, observable} from 'mobx';
+import _ from 'lodash';
+
+export interface WindowSize {
+    width: number,
+    height: number
+}
+
+class WindowStore {
+    constructor() {
+        if (typeof window === 'object') {
+            this.window = window;
+            this.getWindowSize();
+            this.window.addEventListener("resize", this.handleWindowResize)
+        }
+    }
+
+    @action
+    private getWindowSize() {
```

Comment:
this probably needs to be bound (@auto-bind) to work.  my guess is that's what's wrong.